src/main/java/ch/njol/skript/expressions/ExprDurability.java

```diff
@@ -36,78 +36,80 @@
 import ch.njol.skript.util.slot.Slot;
 import ch.njol.util.coll.CollectionUtils;
 
-/**
- * @author Peter Güttinger
- */
-@Name("Data/Damage Value")
-@Description({"The data/damage value of an item/block. Data values of blocks are only supported on 1.12.2 and below.",
-		"You usually don't need this expression as you can check and set items with aliases easily, ",
-		"but this expression can e.g. be used to \"add 1 to data of &lt;item&gt;\", e.g. for cycling through all wool colours."})
-@Examples({"set damage value of player's tool to 10",
-		"set data value of target block of player to 3",
-		"add 1 to the data value of the clicked block",
-		"reset data value of block at player"})
-@Since("1.2")
+@Name("Damage Value/Durability")
+@Description("The damage value/durability of an item.")
+@Examples({
+	"set damage value of player's tool to 10",
+	"reset the durability of {_item}",
+	"set durability of player's held item to 0"
+})
+@Since("1.2, INSERT VERSION (durability reversed)")
 public class ExprDurability extends SimplePropertyExpression<Object, Long> {
 
+	private boolean durability;
+
 	static {
-		register(ExprDurability.class, Long.class, "((data|damage)[s] [value[s]]|durabilit(y|ies))", "itemtypes/blocks/slots");
+		register(ExprDurability.class, Long.class, "(damage[s] [value[s]]|durability:durabilit(y|ies))", "itemtypes/slots");
```

Comment:
I'm not sure that slots need to be explicitly supported - there is a Slot -> ItemType converter.

Maybe remove it and see if it still works ok. If it does work okay, I think you can just switch to
ItemType#getItemMeta and ItemType#setItemMeta

Reply:
It doesn't update the item if it was a slot :shrug:

Reply:
the slots need to be manually updated yeah - my b